packages/core/__tests__/fetchEntityConfiguration.test.ts

```diff
@@ -0,0 +1,78 @@
+import { describe, it } from 'node:test'
+import { createEntityConfiguration, fetchEntityConfiguration } from '../src/entityConfiguration'
+import type { JsonWebKey } from '../src/jsonWeb'
+import type { SignCallback, VerifyCallback } from '../src/utils'
+
+import assert from 'node:assert/strict'
+import nock from 'nock'
```

Comment:
I've had issues with nock in Paradym when using native node fetch, so I moved to `msw` which works quite well. If you're not experiencing these issues, nock is fine with me, as the API is nice 👍

Reply:
nock beta supports the native fetch! If I encounter errors with it I'll swap it.